### PR TITLE
Address some possible inconsistencies in 1.2.9

### PR DESCRIPTION
**Pull request by vsemozhetbyt**

### CODE REVIEW

File: 1-js/02-first-steps/09-comparison/article.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ In JavaScript they are written like this:
 - Equals: `a == b`, please note the double equality sign `==` means the equality test, while a single one `a = b` means an assignment.
 - Not equals. In maths the notation is <code>&ne;</code>, but in JavaScript it's written as <code>a != b</code>.
 
-In this article we'll learn more about different types of comparisons, how JavaScript makes them, including important peculiarities. 
+In this article we'll learn more about different types of comparisons, how JavaScript makes them, including important peculiarities.
```
[the remark this answers is not in the document]
Comment (vsemozhetbyt) [reply 1-js/02-first-steps/09-comparison/article.md]: Thanks.

**Comment by vsemozhetbyt** [comment]
Sorry, ny finger slipped so the second commit title is a bit off)

File: 1-js/02-first-steps/09-comparison/article.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ From JavaScript's standpoint, this result is quite normal. An equality check con
 
 ## Strict equality
 
-A regular equality check `==` has a problem. It cannot differentiate `0` from `false`:
+A regular equality check `==` has a problem. For example, it cannot differentiate `0` from `false`:
```
Comment (iliakan) [inline 1-js/02-first-steps/09-comparison/article.md:115]: Please remove "for example", it doesn't fit the phrase.

**Review by iliakan — CHANGES_REQUESTED** [review]
Please remove "for example", it doesn't fit the phrase.

File: 1-js/02-first-steps/09-comparison/article.md (hunk 2 above)
Comment (vsemozhetbyt) [reply 1-js/02-first-steps/09-comparison/article.md:115]: I will soon. But from the current wording, would not a reader think the 0\false issue is the only issue with ==?

Comment (iliakan) [reply 1-js/02-first-steps/09-comparison/article.md:115]: We're talking about this particular problem here. 

Maybe you'd want to further modify the statement. Right now, "For example" just doesn't fit.

Comment (vsemozhetbyt) [reply 1-js/02-first-steps/09-comparison/article.md:115]: Ok.

Comment (vsemozhetbyt) [reply 1-js/02-first-steps/09-comparison/article.md:115]: Ok.

Comment (vsemozhetbyt) [reply 1-js/02-first-steps/09-comparison/article.md:115]: Ok.

Comment (MuhammedZakir) [reply 1-js/02-first-steps/09-comparison/article.md:115]: https://www.dictionary.com/browse/for-example :
> This program has problems—for instance, it's hard to retrieve lost data.

"For instance" and "for example" can be used interchangeably. But "for instance" may sound better in this case.

Comment (vsemozhetbyt) [reply 1-js/02-first-steps/09-comparison/article.md]: Change excluded,

**Pull request merged**